### ISSUE TITLE
Improve unit test error messages

### ISSUE THREAD

**Issue by Flamefire** [easybuilders/easybuild-easyconfigs#9542]
I suggest to rework the assert messages for pretty much all tests. The current ones are e.g. "sanity_pip_check is enabled in X". So if you see the test failing on travis this will be what is printed. 

The problem: The opposite is the case: The check failed because it is not enabled. This made me loose some time chasing for the failure because I couldn't understand the message.

Better: "sanity_pip_check should be enabled in X" or "sanity_pip_check is not enabled in X"

**Comment by boegel** [comment]
I understand the confusion, we can indeed do a better job here...

It should definitely be `sanity_pip_check should be enabled in X` in this specific case though, since that is exactly what is being asserted. Probably `must be enabled` is more correct.

**Comment by verdurin** [comment]
I agree with @Flamefire that the messages as currently worded are counter-intuitive.

Do you have a PR to fix this?

**Comment by Flamefire** [comment]
No and it's likely a bit of work. I'd replace `failing_checks` by `failures` in e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/be8305a7b896e6f01af7af5ab61f19990a9bd07a/test/easyconfigs/easyconfigs.py#L568 and adjust the wording to the negative (like `"'download_dep_fail' not set in %s" % ec_fn`)
